### ISSUE TITLE
Cannot add custom file to after/ directory.

### ISSUE THREAD

**Issue by MisterSnrub** [VundleVim/Vundle.vim#805]
Before I installed Vundle, I had a file in my standard .vim/after/ftplugin/ directory called perl.vim, which contained the following code:

    " Enable here documentation for Perl files
    " For more info, see:
    " http://stackoverflow.com/questions/6864420/change-filetype-or-other-vim-settings-blockwise-in-a-file
    
    syntax on
    
    if exists("b:current_syntax")
      unlet b:current_syntax
    endif
    
    syntax include @CSS syntax/css.vim
    syntax region cssSnip matchgroup=Snip start=+<<['"]CSS['"].*;\s*$+ end=+^\s*CSS$+ contains=@CSS
    
    if exists("b:current_syntax")
      unlet b:current_syntax
    endif
    
    syntax include @XML syntax/xml.vim
    syntax region xmlSnip matchgroup=Snip start=:<<\(['"]\?\)XML\1\s*;\s*$: end=:^\s*XML$: contains=@XML
    
    if exists("b:current_syntax")
      unlet b:current_syntax
    endif
    
    syntax include @SQL syntax/sql.vim
    syntax region sqlSnip matchgroup=Snip start=:<<\(['"]\?\)SQL\1\s*;\s*$: end=:^\s*SQL$: contains=@SQL
    
    if exists("b:current_syntax")
      unlet b:current_syntax
    endif
    
    syntax include @HTML syntax/html.vim
    syntax region htmlSnip matchgroup=Snip start=:<<\(['"]\?\)HTML\1\s*;\s*$: end=:^\s*HTML$: contains=@HTML
    
    if exists("b:current_syntax")
      unlet b:current_syntax
    endif
    
    syntax include @JAVASCRIPT syntax/javascript.vim
    syntax region javascriptSnip start=+<<['"]JAVASCRIPT['"].*;\s*$+ end=+^\s*JAVASCRIPT$+ contains=@JAVASCRIPT
    
    hi link Snip Special

Basically it will format the syntax inside of a here-doc for a Perl file, based on what boundary text you use. Normally this file will go into the ~/.vim/after/ftplugin/ folder, but now that I've installed Vundle, I don't know where to put this file. It is called perl.vim, and it is only supposed to be run when editing perl files.

**Comment by ryanoasis** [comment]
Hmm I wouldn't think that'd be a problem, can you see if info on #301 is of any help?
